src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,207 @@
 package core.basesyntax;
 
+import java.util.Collection;
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
fields have default values

Reply:
Changed